examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -411,3 +411,29 @@ class SubscribeEvent : public SubscribeCommand
     chip::Optional<bool> mKeepSubscriptions;
     chip::Optional<std::vector<bool>> mIsUrgents;
 };
+
+class ReadAll : public ReadCommand
```

Comment:
Is it intentionally part of this commit?

Reply:
That is a wrong rebase! Thanks.